tests/test_oauth2_message.py

```diff
@@ -30,9 +30,35 @@
 from oic.oauth2.message import json_deserializer
 from oic.oauth2.message import json_serializer
 from oic.oauth2.message import sp_sep_list_deserializer
+from oic.utils.keyio import build_keyjar
 
 __author__ = 'rohe0002'
 
+keys = [
```

Comment:
All this setup should probably be in one or more pytest.fixture functions.

It clearly is a test fixture.